contrib/pg_tde/documentation/docs/global-key-provider-configuration/set-principal-key.md

```diff
@@ -0,0 +1,37 @@
+# Global Principal Key Configuration
+
+You can configure a default principal key using a global key provider. This key will be used by all databases that do not have their own encryption keys configured.
+
+## Create a Default Principal Key
```

Comment:
You have different style for titles: Here's the Headline style while the next section has Sentence style. It makes sense to align it

Reply:
Throughout the whole documentation.